uiowa/uiowa
# Update smart_date

Resolves https://github.com/uiowa/uiowa/issues/5664
Relates to https://github.com/uiowa/uiowa/issues/6381

# To Test

```
ddev composer install && ddev blt ds --site=grad.uiowa.edu
```

- Test out the date range option on UI Events block in v3.
- Confirm that thesis defense date functionality still works as expected for grad.uiowa.edu (duration 0 shows just like prod on the form, during save, there are no errors)

https://git.drupalcode.org/project/smart_date/-/compare/3.6.1...3.7.2?from_project_id=59941&straight=false
